### ISSUE TITLE
Unable to interact with deploy chart release

### ISSUE THREAD

**Issue by aidan-melen** [lensapp/lens#2346]
**Describe the bug**
Cannot interact with [kube-prometheus-stack](https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack) release in helm. When I select the release, the details page hangs forever. 

I think this is an issue with Lens because I can interact with the release using the `helm` commandline tool i.e. retrieve values, upgrade, etc.

**To Reproduce**
1. deploy release
```
helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
helm repo update
helm install [RELEASE_NAME] prometheus-community/kube-prometheus-stack
```
2. once deployed, navigate to Apps > Releases in Lens and select the release.
3. Then the Release details side pane with load forever.

**Expected behavior**
I expect to be able to interact with the release just like I can using the `helm` command line tool.

**Screenshots**
![Screen Shot 2021-03-16 at 12 33 37 PM](https://user-images.githubusercontent.com/54326778/111362126-0db0d680-8654-11eb-867a-e355335dc957.png)

Here we can see that I can upgrade the release from the command line just fine (ruling out the chart from the problem).

![Screen Shot 2021-03-16 at 12 43 00 PM](https://user-images.githubusercontent.com/54326778/111363137-48673e80-8655-11eb-84d8-f27192b4fddd.png)

**Environment (please complete the following information):**
- Lens Version:
```
Lens: 4.1.4
Electron: 9.4.0
Chrome: 83.0.4103.122
Node: 12.14.1
Copyright 2020 Mirantis, Inc.
```
- OS: OSX
- Installation method (e.g. snap or AppImage in Linux): Homebrew

**Kubeconfig:**
The kubeconfig is good because I can interact with other releases in the same cluster. I just can't interact with the kube-prometheus-stack release.

**Comment by jim-docker** [comment]
Hi, is there RBAC on this cluster? To see Helm releases you need to be able to get/list secrets. (Regardless, the spinner should not spin forever, but there is a fix coming for that. Similar to #2239?

**Comment by aidan-melen** [comment]
@jim-docker The user I am authenticated with is a member of the `system:masters` group. So likely not getting blocked by RBAC.

Unlike #2239 I am able to like the release, I just can use Lens to upgrade the Release because I get blocked by the spinner wheel.

I noticed that other ticket contained logs. Where do I find Lens logs on MacOs?

**Comment by jim-docker** [comment]
https://github.com/lensapp/lens/issues/2239#issuecomment-788062900

I believe the spinner issue has been fixed, and it may be in 4.2, not sure if it made it (there is a release candidate available you could try https://github.com/lensapp/lens/releases/tag/v4.2.0-rc.1).

**Comment by LucaDev** [comment]
Same Issue with Lens 5 Beta 4. I'm in the cluster-admin group. I dont think this is a rbac issue. Im able to edit the traefik release without any problems.
![image](https://user-images.githubusercontent.com/5812715/118701411-fb017c00-b813-11eb-8b58-ee8e33169920.png)

Console Error:
![image](https://user-images.githubusercontent.com/5812715/118701569-25533980-b814-11eb-8efe-90c3a29b6ec0.png)

**Comment by Nokel81** [comment]
@LucaDev This should be fixed by https://github.com/lensapp/lens/pull/2924 as well.

**Comment by LucaDev** [comment]
@Nokel81 now that https://github.com/lensapp/lens/pull/2924 is merged, this can be closed, right?

**Comment by Nokel81** [comment]
Yes. Fixed by #2924. Please open a new issue if this continues to be a problem in v5.
